project.mml

```diff
@@ -1615,9 +1616,9 @@ Layer:
           name,
           layer,
           COALESCE(
-           'highway_' || CASE WHEN tags @> 'ford=>yes' OR tags @> 'ford=>stepping_stones' THEN 'ford' END,
-           'leisure_' || CASE WHEN leisure IN ('slipway', 'track') THEN leisure END,
-           'attraction_' || CASE WHEN tags @> 'attraction=>water_slide' THEN 'water_slide' END
+           'highway_' || CASE WHEN tags @> 'ford=>yes' OR tags @> 'ford=>stepping_stones' THEN 'ford' ELSE NULL END,
```

Comment:
"ELSE NULL" has accidentally been added on these three lines, please remove

Reply:
fixed

Reply:
Only the first line was fixed